CONTRIBUTING.md

```diff
@@ -80,6 +80,11 @@ There are two ways of running the `goat` locally:
     Note: Linter env variables must be formatted as <LINTER_CONFIG>, i.e. RUFF_CONFIG, CFN_LINT_CONFIG,
     etc., and the values supplied will overwrite the default arguments supplied in the goat.
 
+4. Autofix is available for certain linters and can be invoked in two ways:
+   1. `docker run -e AUTO_FIX="true" -v $PWD:/goat/ -rm <hash>`
+   2. `pipenv run invoke reformat`
+
 ### Note
```

Comment:
This is becoming a linter updates section or similar. Make heading more precise based on contents please

Reply:
Done